### PR TITLE
feat: add Dex user interfaces docs

### PR DESCRIPTION
**Pull request by dib542**
This adds a new short page to explain in general the new Orderbook UI of the Neutron Dex on the Neutron app (app.neutron.org).

It includes a mention of the usage of TradingView software as a part of our licence agreement to use the TradingVIew chart on the Orderbook page.

This PR also includes a small update to make the message headings on the `/neutron/modules/dex/messages` page consistent (other H2 "##" level headings do not use the word "message")

### CODE REVIEW

**Review by jcompagni10 — APPROVED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Comment by dib542** [comment]
This should be modified to point to Onboarding app address, as this app will be Neutron branded and contain a TradingView chart in the Orderbook page.

tracking in: https://hadronlabs.atlassian.net/browse/UI-858

**Comment by dib542** [comment]
☝️ added modifications to reference new Orderbook page and Onboarding app UI at app.neutron.org.

File: docs/neutron/modules/dex/user-interfaces.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+You can use Neutron DEX by directly crafting [DEX messages](/neutron/modules/dex/messages) and sending them through transactions in your own scripts, but you can also use various user interfaces to interact with the DEX.
+
+We are working with several partners in the Neutron [ecosystem](https://app.neutron.org/ecosystem) (such as [Astroport](https://app.astroport.fi)) that use Neutron DEX and by using these applications you can benefit from the DEX's features. We also provide a UI as described here in the Neutron app for placing orders on the DEX.
+
+## Neutron app
+
+### Orderbook
+
+The Neutron app orderbook is available at [app.neutron.org/orderbook](https://app.neutron.org/orderbook)
```
Comment (sotnikov-s) [inline docs/neutron/modules/dex/user-interfaces.md:11]: when will the page be live? can only see the Bridge page now

**Review by sotnikov-s — APPROVED** [review]
the rest LGTM

File: docs/neutron/modules/dex/user-interfaces.md (hunk 1 above)
Comment (dib542) [reply docs/neutron/modules/dex/user-interfaces.md:11]: "Soon". The page is behind a flag until release. Targeting release the 16th.

Comment (dib542) [reply docs/neutron/modules/dex/user-interfaces.md:11]: @sotnikov-s the orderbook has finally been released at https://app.neutron.org/orderbook so this PR is the documentation for that

File: docs/neutron/modules/dex/user-interfaces.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# User Interfaces
+
+You can use Neutron DEX by directly crafting [DEX messages](/neutron/modules/dex/messages) and sending them through transactions in your own scripts, but you can also use various user interfaces to interact with the DEX.
+
+We are working with several partners in the Neutron [ecosystem](https://app.neutron.org/ecosystem) (such as [Astroport](https://app.astroport.fi)) that use Neutron DEX and by using these applications you can benefit from the DEX's features. We also provide a UI as described here in the Neutron app for placing orders on the DEX.
```
Comment (jcompagni10) [inline docs/neutron/modules/dex/user-interfaces.md:5]: Astroport is not using Dex yet. Maybe hold off on this part

**Review by jcompagni10 — DISMISSED** [review]
LGTM

File: docs/neutron/modules/dex/user-interfaces.md (hunk 2 above)
Comment (dib542) [reply docs/neutron/modules/dex/user-interfaces.md]: Removed in [5f5afeb](https://github.com/neutron-org/neutron-docs/pull/170/commits/5f5afeb2f4141a74a5ff0b91959e1973cb0053dd) (pre re-review 🙏 )

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**